### DISCUSSION TITLE
Getting the angle at which the path ray has interacted with the material

### DISCUSSION THREAD

**Discussion by itsghaniz** [NVlabs/sionna#544] [Q&A]
Hi, Hope everyone is doing well. I wanted to find if there is a way to find the angle at which a ray has interacted with a material. is this possible. I know that path.vertices is used to find the coordinates and path. objects to find the object id, but how to find at the vertice, of the object what is angle of arrival and angle of departure relative to the object. 
thanks in advance

**Comment by SebastianCa** [comment]
Hi @itsghaniz,
there seems to be an overlap with #505. Please continue the discussion there.
